docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,436 @@
+    bytes proof = 2;
+}
+```
+If not provided during submission, it must be submitted later using the 
+`MsgAddBTCDelegationInclusionProof` message.
+* `slashing_tx`: 
+The slashing transaction in hex format. Generated during both staking and 
+unbonding phases, this transaction spends the staking or unbonding transaction 
+in cases of protocol violations such as double-signing. To see exactly what 
+constitutes a valid slashing transaction, see the 
+[`Slashing Transaction` section](./staking-script.md#slashing-transaction).
+A valid signature for this transaction must be included in the 
+`slashing_tx_sig` field.
+To generate or obtain the slashing transaction, you can use the libraries 
+mentioned below:
+  * Using our [Golang BTC staking library](../btcstaking)
+  * Using our [TypeScript BTC staking library](https://github.com/babylonlabs-io/btc-staking-ts)
+  * Implementing the transaction generation themselves using the 
+    [staking script specification](./staking-script.md) as a reference.
+* `delegator_slashing_sig`: 
+The signature for the `slashing_tx`, which is crucial for authorizing the 
+transaction in the event of a protocol violation, such as double-signing. 
+The signature, generated with the private key for the staker's secp256k1 
+public key (`btc_pk`), should be compatible with ECDSA or Schnorr (BIP340).
+* `unbonding_time`: 
+The timelock period for unbonding, measured in Bitcoin blocks, indicating 
+how long funds remain locked. It should be an unsigned integer (uint32) and 
+ensure that any `unbonding_time` you choose is more than that specified 
+by the `min_unbonding_time_blocks` parameter in the genesis file.
+* `unbonding_tx`: 
+The unbonding transaction in hex format. An unbonding transaction allows a BTC 
+staker to unlock their funds by obtaining necessary signatures and broadcasting 
+the transaction to the Bitcoin network.
+Details on what constitutes a valid unbonding transaction can be found
+[here](./staking-script.md#unbonding-transaction). Once confirmed, the staker 
+regains access to their BTC.
+  * Using our [Golang BTC staking library](../btcstaking)
+  * Using our [TypeScript BTC staking library](https://github.com/babylonlabs-io/btc-staking-ts)
+  * Implementing the transaction generation themselves using the 
+    [staking script specification](./staking-script.md) as a reference.
+* `unbonding_value`:
+The staker will set this amount to what they wish the unbonded amount should be.
+This should be specified as an integer, representing the amount of Bitcoin in 
+satoshis. Since 1 Bitcoin equals 100,000,000 satoshis, this value should be a 
+whole number without decimals.
+* `unbonding_slashing_tx`: 
+The unbonding slashing transaction in hex format. This transaction spends the 
+unbonding transaction in cases of protocol violations such as double-signing.  
+To genertate or obtain the slashing transaction, you can use the libraries 
```

Comment:
We should not that this is a schnorr signature in hex format, like with the `delegator_slashing_sig`.